openapi.yaml

```diff
@@ -7108,12 +7081,17 @@ components:
           type: string
           description: Only applicable to customers with the Billing Cost Centre Module enabled. Allows the user to set a billing cost centre for the Workspace.
           example: Hollywood
-      xml:
-        name: workspace
-        prefix: ns3
-      x-tags:
-        - Workspaces
-      title: Workspace
+        link:
```

Comment:
Updated 'link' attribute from workspace model to be referencing Link model object